src/Security.tsx

```diff
@@ -43,13 +41,6 @@ const Security: React.FC<{
       return;
     }
 
-    // Add default restoreOriginalUri callback
-    if (!oktaAuth.options.restoreOriginalUri) {
```

Comment:
As users should have control on configs of oktaAuth instance, we can do check here and provide proper warning if two restoreOriginalUri are provided (one from Security prop, one from oktaAuth options).

Reply:
@denysoblohin-okta ^ thoughts on the comment above?

Reply:
@shuowu Updated PR